pandas/api/extensions/__init__.py

```diff
@@ -19,6 +19,7 @@
     ExtensionArray,
     ExtensionScalarOpsMixin,
 )
+from pandas.core.arrays._mixins import NDArrayBackedExtensionArray
```

Comment:
i am shocked we don't have a tests that asserts this file e.g. `test_api.py` can you add one ?

Reply:
we should probably import this into `pandas.core.arrays.__init__` and here do the import `from pandas.core.arrays`